src/python/module/smartredis/dataset.py

```diff
@@ -35,7 +35,7 @@
 
 class Dataset(SRObject):
     def __init__(self, name: str) -> None:
-        """Initialize a Dataset object
+        """Initialize a Dataset object in-memory
```

Comment:
revert this change as it suggests that there is a way to initialize a dataset out of memory